rooms-quickstart/client.js

```diff
@@ -0,0 +1,98 @@
+// <Create a rooms client>
+const { RoomsClient } = require("@azure/communication-rooms");
+const { CommunicationIdentityClient } = require("@azure/communication-identity");
+
+export async function main() {
+  const connectionString = "<insert-connecction-string>";
+  const identityClient = new CommunicationIdentityClient(connectionString);
+  const user1 = await identityClient.createUserAndToken(["voip"]);
+  const user2 = await identityClient.createUserAndToken(["voip"]);
+
+  // create RoomsClient
+  const roomsClient = new RoomsClient(connectionString);
+
+  var validFrom = new Date(Date.now());
+  var validUntil = new Date(validFrom.getTime() + 5 * 60 * 1000);
+
+  // options payload to create a room
+  const createRoomOptions = {
+    validFrom: validFrom,
+    validUntil: validUntil,
+    participants: [
+      {
+        id: user1.user,
+        role: "Attendee",
+      },
+    ],
+  };
+
+  // create a room with the request payload
+  const createRoom = await roomsClient.createRoom(createRoomOptions);
+  const roomId = createRoom.id;
+  console.log(`Created Room`);
+
+  // retrieves the room with corresponding ID
+  const getRoom = await roomsClient.getRoom(roomId);
+  console.log(`Retrieved Room with ID ${roomId}`);
+  
+  validFrom.setTime(validUntil.getTime());
+  validUntil.setTime(validFrom.getTime() + 5 * 60 * 1000);
+
+  // request payload to update a room
+  const updateRoomRequest = {
```

Comment:
Might want to change this to updateRoomOptions to stay consistent with createRoomOptions above